homeassistant/components/modern_forms/light.py

```diff
@@ -0,0 +1,143 @@
+"""Support for Modern Forms Fan lights."""
+from __future__ import annotations
+
+from typing import Any
+
+from aiomodernforms.const import LIGHT_POWER_OFF, LIGHT_POWER_ON
+import voluptuous as vol
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS,
```

Comment:
This is deprecated, use `COLOR_MODE_BRIGHTNESS` instead.
```suggestion
    COLOR_MODE_BRIGHTNESS
```

Reply:
done